files/qsub.pl

```diff
@@ -583,23 +583,33 @@ sub parse_script
         }
     };
 
-    # handle queues
-    if ($orig_argsh{'q'} || $set{'q'}) {
-        warn "Please do not set a queue (\'-q\' option or \'#PBS -q\' directive).\n" .
-            "Right now it sets only the default walltime based on the queue,\n" .
-            "So please request explicitly the desired walltime.\n";
-
-    }
+    # handle queues and special queues (as a partition)
     my $userqueue = $destination || $set{'q'};
     if ($userqueue) {
+        if ($userqueue eq "special" ) {
+            if (defined $ENV{SLURM_CLUSTERS} && $ENV{SLURM_CLUSTERS} !~ m/[;:,\/]/)  {
+                push(@cmd, '--partition', "$ENV{SLURM_CLUSTERS}_$userqueue");
+            } else {
+                fatal("You have problem with your environment!\n" .
```

Comment:
it's not really a problem, we just don't support it. so rephrase this

Reply:
Done!